src/onInit/addVariables.js

```diff
@@ -1,11 +1,20 @@
 export default function addVariables() {
+    const ordinalTimeSettings = this.config.time_cols
+        .filter(time_col => time_col.type === 'ordinal')
+        .pop();
```

Comment:
`array.pop()` instead of `array[0]` is cute. Probably a bit easier to read.

Reply:
Yeah, but [`array.find()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find) is the best option, but the one time I tried it out babel didn't add a polyfill so it failed in IE.